app/views/posts/show.html.erb

```diff
@@ -12,4 +16,11 @@
       <% end %>
     </ul>
   </div>
-</ul>
\ No newline at end of file
+
+  <div class="addNewdiv">
+    <%= link_to "comment", new_user_post_comment_path(user_id: @post.user_id, post_id: @post.id ), class: "addCom" %>
+    <%= form_with model: @like, url: user_post_likes_path(post_id: @post.id) do |form| %>
+      <%= form.submit "Like post", class: "addCom" %>
+    <% end %>
```

Comment:
- [ ] As requested by the previous reviewers, Kindly create an action method that enables users to like a post. We noticed you don't have a likes controller, kindly create one with an action method that enables users to like a post (def create). You don't need the action method `new` because liking a post should be automatic after clicking a button, you only need the `create` action method.